parsl/tests/configs/cooley_ssh-il_single_node.py

```diff
@@ -1,40 +1,30 @@
-import pytest
-from parsl.tests.utils import get_rundir
+from libsubmit.channels.ssh_il.ssh_il import SSHInteractiveLoginChannel
+from libsubmit.providers.cobalt.cobalt import Cobalt
+from parsl.config import Config
+from parsl.executors.ipp import IPyParallelExecutor
 from parsl.tests.user_opts import user_opts
+from parsl.tests.utils import get_rundir
 
-if 'cooley' not in user_opts:
-    pytest.skip('cooley user_opts not configured', allow_module_level=True)
-else:
-    info = user_opts['cooley']
+config = Config(
+    executors=[
+        IPyParallelExecutor(
+            label='cooley_ssh_il_local_single_node',
+            provider=Cobalt(
+                channel=SSHInteractiveLoginChannel(
+                    hostname='cooleylogin1.alcf.anl.gov',
+                    username=user_opts['cooley']['username'],
+                    script_dir="/home/{}/parsl_scripts/".format(user_opts['cooley']['username'])
+                ),
+                nodes_per_block=1,
+                tasks_per_node=1,
+                init_blocks=1,
+                max_blocks=1,
+                walltime="00:05:00",
```

Comment:
Please check to make sure we are not missing `account` or `partition` options for cobalt